packages/edge/src/edge-headers.ts

```diff
@@ -68,6 +74,22 @@ function getHeader(request: Request, key: string): string | undefined {
   return request.headers.get(key) ?? undefined;
 }
 
+/**
+ * Converts the 2 digit countryCode into a flag emoji by adding the current character value to the emoji flag unicode starting position.
+ *
+ * @param countryCode The country code returned by: `getHeader(request, COUNTRY_HEADER_NAME)`.
+ * @returns A flag emoji or undefined.
+ */
+function getFlag(countryCode: string | undefined): string | undefined {
+  const regex = new RegExp('^[A-Z]{2}$').test(countryCode!);
+  if (!countryCode || !regex) return undefined;
+  return String.fromCodePoint(
+    ...countryCode
+      .split('')
+      .map(char => EMOJI_FLAG_UNICODE_STARTING_POSITION + char.charCodeAt(0))
+  );
+}
```

Comment:
I saw this code on a blog post recently (cc @Kikobeats who sent it), do we need to provide attribution?

Reply:
Yes, it's this one: https://dev.to/jorik/country-code-to-flag-emoji-a21

I can add attribution, no problem.